remo/profiles/apps.py

```diff
@@ -11,8 +11,9 @@ def profiles_set_groups(sender, **kwargs):
         return True
 
     perms = {'create_user': ['Admin', 'Mentor'],
-             'can_edit_profiles': ['Admin'],
-             'can_delete_profiles': ['Admin']}
+             'can_edit_profiles': ['Admin', 'Council', 'Peers'],
```

Comment:
Let's drop Peers from here. The less people have edit access here the better.

Reply:
You're right. Will do.